Formula/anime-downloader.rb

```diff
@@ -19,9 +19,9 @@ class AnimeDownloader < Formula
   end
 
   depends_on "aria2"
+  depends_on "libpython-tabulate"
   depends_on "node"
```

Comment:
Audit failure may be due to `node` which uses `python` alias that is currently `python@3.9`. I believe this is just a runtime dependency needing `python3` on PATH, so there are some instances where we use allowlist (e.g. `emscripten`)